cortex-cms/cortex-plugins-core
# Resolve CKEditor Issues & Upgrade

* Upgrades to latest CKEditor
* Insures CKEditor is instantiated on page load, with eventual support for Turbolinks built in
* Add Source button to CKEditor
* Fix missing icon for `insertMedia` CKEditor plugin
* Fix straggling old `cortex-field_types-core` naming
* Ensure CKEditor is precompiled
